v03_pipeline/lib/reference_data/dataset_table_operations.py

```diff
@@ -36,10 +35,7 @@ def update_or_create_joined_ht(
             continue
 
         # Join the new one!
-        hl._set_flags(use_new_shuffle=None, no_whole_stage_codegen='1')  # noqa: SLF001
```

Comment:
Were you able to find a different solution to the shuffle problem?

Reply:
eh... I think it's wrong to remove it actually, good call!  Instead I should have moved it to the clinvar file. 🤦